### ISSUE TITLE
[Bug]: Unable to apply the Post Install Patch

### ISSUE THREAD

**Issue by Tiger-AC** [dortania/OpenCore-Legacy-Patcher#549]
### Machine Model

iMac12,2

### Application Version

Latest Build (nightly)

### Application Variant

TUI (Text User Interface)

### What versions of macOS are you seeing the problem on?

macOS 11, Big Sur

### Where does this issue happen?

Within macOS (After Logging in)

### What is the Isssue?

Tried to apply the latest nightly build TUI to my iMac12,2 already running the latest OCLP (v030 nightly, downloaded, installed, and restarted a few minutes ago) Big Sur 11.6, which is quite stable, much appreciated for the excellent work. The initial Post Install Patch was applied a week ago with the latest v030n at the time of installation.
Please find the info in the additional info for the issue.

### Any Additional Information

- Downloading package
- 92.5893144607544MB Downloaded
- Download completed
- Unzipping download...
- Renaming folder
- Binaries downloaded to:
/private/var/folders/bx/51cg1zdd6bbdmknb5xvs1r5r0000gn/T/_MEIudwvJz/payloads
Press [ENTER] to continue
- Found Root Volume at: disk4s5
- Mounting APFS Snapshot as writable
Password:
- Successfully mounted the Root Volume
- Running patches for iMac12,2
- Installing legacy TeraScale 2 Patches
- Detected supported OS, installing Acceleration Patches
- Installing TeraScale 2 Acceleration Kext patches for Big Sur
- Couldn't find AMDRadeonX4000.kext, skipping
- Couldn't find AMDRadeonX4000HWServices.kext, skipping
- Couldn't find AMDRadeonX5000.kext, skipping
- Couldn't find AMDRadeonX5000HWServices.kext, skipping
- Couldn't find AMDRadeonX6000.kext, skipping
- Couldn't find AMDRadeonX6000Framebuffer.kext, skipping
- Couldn't find AMDRadeonX6000HWServices.kext, skipping
- Couldn't find AMD7000Controller.kext, skipping
- Couldn't find AMD8000Controller.kext, skipping
- Couldn't find AMD9000Controller.kext, skipping
- Couldn't find AMD9500Controller.kext, skipping
- Couldn't find AMD10000Controller.kext, skipping
- Couldn't find AppleIntelBDWGraphics.kext, skipping
- Couldn't find AppleIntelBDWGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelCFLGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelHD4000Graphics.kext, skipping
- Couldn't find AppleIntelHD5000Graphics.kext, skipping
- Couldn't find AppleIntelICLGraphics.kext, skipping
- Couldn't find AppleIntelICLLPGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelKBLGraphics.kext, skipping
- Couldn't find AppleIntelKBLGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelSKLGraphics.kext, skipping
- Couldn't find AppleIntelSKLGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelFramebufferAzul.kext, skipping
- Couldn't find AppleIntelFramebufferCapri.kext, skipping
- Couldn't find AppleParavirtGPU.kext, skipping
- Couldn't find GeForce.kext, skipping
- Couldn't find IOGPUFamily.kext, skipping
- Couldn't find AppleCameraInterface.kext, skipping
- Installing General Acceleration Kext patches for Big Sur/Monterey
- Found conflicting kext, Deleting Root Volume's IOAcceleratorFamily2.kext
- Adding IOAcceleratorFamily2.kext
- Found conflicting kext, Deleting Root Volume's IOSurface.kext
- Adding IOSurface.kext
- Found conflicting kext, Deleting Root Volume's AMD2400Controller.kext
- Adding AMD2400Controller.kext
- Found conflicting kext, Deleting Root Volume's AMD2600Controller.kext
- Adding AMD2600Controller.kext
- Found conflicting kext, Deleting Root Volume's AMD3800Controller.kext
- Adding AMD3800Controller.kext
- Found conflicting kext, Deleting Root Volume's AMD4600Controller.kext
- Adding AMD4600Controller.kext
- Found conflicting kext, Deleting Root Volume's AMD4800Controller.kext
- Adding AMD4800Controller.kext
- Found conflicting kext, Deleting Root Volume's AMD5000Controller.kext
- Adding AMD5000Controller.kext
- Found conflicting kext, Deleting Root Volume's AMD6000Controller.kext
- Adding AMD6000Controller.kext
- Found conflicting kext, Deleting Root Volume's AMDFramebuffer.kext
- Adding AMDFramebuffer.kext
- Found conflicting kext, Deleting Root Volume's AMDLegacyFramebuffer.kext
- Adding AMDLegacyFramebuffer.kext
- Found conflicting kext, Deleting Root Volume's AMDLegacySupport.kext
- Adding AMDLegacySupport.kext
- Found conflicting kext, Deleting Root Volume's AMDRadeonVADriver.bundle
- Adding AMDRadeonVADriver.bundle
- Found conflicting kext, Deleting Root Volume's AMDRadeonVADriver2.bundle
- Adding AMDRadeonVADriver2.bundle
- Found conflicting kext, Deleting Root Volume's AMDRadeonX3000.kext
- Adding AMDRadeonX3000.kext
- Found conflicting kext, Deleting Root Volume's AMDRadeonX3000GLDriver.bundle
- Adding AMDRadeonX3000GLDriver.bundle
- Found conflicting kext, Deleting Root Volume's AMDShared.bundle
- Adding AMDShared.bundle
- Found conflicting kext, Deleting Root Volume's AMDSupport.kext
- Adding AMDSupport.kext
- Found conflicting kext, Deleting Root Volume's ATIRadeonX2000.kext
- Adding ATIRadeonX2000.kext
- Found conflicting kext, Deleting Root Volume's ATIRadeonX2000GA.plugin
- Adding ATIRadeonX2000GA.plugin
- Found conflicting kext, Deleting Root Volume's ATIRadeonX2000GLDriver.bundle
- Adding ATIRadeonX2000GLDriver.bundle
- Found conflicting kext, Deleting Root Volume's ATIRadeonX2000VADriver.bundle
- Adding ATIRadeonX2000VADriver.bundle
- Installing legacy Sandy Bridge Patches
- Detected supported OS, installing Acceleration Patches
- Installing Sandy Bridge Acceleration Kext patches for Big Sur/Monterey
- Couldn't find AMDRadeonX4000.kext, skipping
- Couldn't find AMDRadeonX4000HWServices.kext, skipping
- Couldn't find AMDRadeonX5000.kext, skipping
- Couldn't find AMDRadeonX5000HWServices.kext, skipping
- Couldn't find AMDRadeonX6000.kext, skipping
- Couldn't find AMDRadeonX6000Framebuffer.kext, skipping
- Couldn't find AMDRadeonX6000HWServices.kext, skipping
- Couldn't find AppleIntelBDWGraphics.kext, skipping
- Couldn't find AppleIntelBDWGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelCFLGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelHD4000Graphics.kext, skipping
- Couldn't find AppleIntelHD5000Graphics.kext, skipping
- Couldn't find AppleIntelICLGraphics.kext, skipping
- Couldn't find AppleIntelICLLPGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelKBLGraphics.kext, skipping
- Couldn't find AppleIntelKBLGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelSKLGraphics.kext, skipping
- Couldn't find AppleIntelSKLGraphicsFramebuffer.kext, skipping
- Couldn't find AppleIntelFramebufferAzul.kext, skipping
- Couldn't find AppleIntelFramebufferCapri.kext, skipping
- Couldn't find AppleParavirtGPU.kext, skipping
- Couldn't find GeForce.kext, skipping
- Deleting IOAcceleratorFamily2.kext
- Couldn't find IOGPUFamily.kext, skipping
- Installing General Acceleration Kext patches for Big Sur/Monterey
- Adding IOAcceleratorFamily2.kext
- Found conflicting kext, Deleting Root Volume's IOSurface.kext
- Adding IOSurface.kext
- Found conflicting kext, Deleting Root Volume's AppleIntelHD3000Graphics.kext
- Adding AppleIntelHD3000Graphics.kext
- Found conflicting kext, Deleting Root Volume's AppleIntelHD3000GraphicsGA.plugin
- Adding AppleIntelHD3000GraphicsGA.plugin
- Found conflicting kext, Deleting Root Volume's AppleIntelHD3000GraphicsGLDriver.bundle
- Adding AppleIntelHD3000GraphicsGLDriver.bundle
- Found conflicting kext, Deleting Root Volume's AppleIntelHD3000GraphicsVADriver.bundle
- Adding AppleIntelHD3000GraphicsVADriver.bundle
- Found conflicting kext, Deleting Root Volume's AppleIntelSNBGraphicsFB.kext
- Adding AppleIntelSNBGraphicsFB.kext
- Found conflicting kext, Deleting Root Volume's AppleIntelSNBVA.bundle
- Adding AppleIntelSNBVA.bundle
- Installing modified AppleIntelSNBGraphicsFB
Traceback (most recent call last):
  File "OpenCore-Patcher.command", line 90, in <module>
  File "OpenCore-Patcher.command", line 19, in __init__
  File "OpenCore-Patcher.command", line 84, in main_menu
  File "utilities.py", line 433, in start
  File "cli_menu.py", line 764, in PatchVolume
  File "sys_patch.py", line 845, in start_patch
  File "sys_patch.py", line 87, in find_mount_root_vol
  File "sys_patch.py", line 534, in patch_root_vol
  File "sys_patch.py", line 399, in gpu_accel_legacy_sandybridge_master
  File "sys_patch.py", line 415, in gpu_accel_legacy_sandybridge_board_id
AttributeError: module 'data.sys_patch_data' has no attribute 'AddIntelGen2AccelPatched'
[623] Failed to execute script 'OpenCore-Patcher' due to unhandled exception!
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]

**Comment by Tiger-AC** [comment]
Thanks for the quick response.
Tried  unpatch with TUI, reboot and patch again with the same result:

- Installing modified AppleIntelSNBGraphicsFB
Traceback (most recent call last):
  File "OpenCore-Patcher.command", line 90, in <module>
  File "OpenCore-Patcher.command", line 19, in __init__
  File "OpenCore-Patcher.command", line 84, in main_menu
  File "utilities.py", line 433, in start
  File "cli_menu.py", line 769, in PatchVolume
  File "sys_patch.py", line 845, in start_patch
  File "sys_patch.py", line 87, in find_mount_root_vol
  File "sys_patch.py", line 534, in patch_root_vol
  File "sys_patch.py", line 399, in gpu_accel_legacy_sandybridge_master
  File "sys_patch.py", line 415, in gpu_accel_legacy_sandybridge_board_id
  File "sys_patch.py", line 274, in add_new_binaries
  File "pathlib.py", line 1072, in __new__
  File "pathlib.py", line 697, in _from_parts
  File "pathlib.py", line 681, in _parse_args
TypeError: expected str, bytes or os.PathLike object, not list
[651] Failed to execute script 'OpenCore-Patcher' due to unhandled exception!

I also tried to inject the AppleIntelSNBGraphicsFB.kext directly into the EFI/OC, and it did not make any difference.
